client/stylesheets/_toCat.scss

```diff
@@ -5594,6 +5594,9 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 5px;
 }
 
-
+textarea {
+    border: 1px solid #d3d3d3 !important;
+    border-radius: 4px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces